assets/stylesheets/global/variables-dark.scss

```diff
@@ -9,7 +9,7 @@ $sidebarMediumWidth: 16rem;
 
 $contentBackground: #2d2d2d;
 
-$textColor: #b4b3af;
+$textColor: #e6e6e5;
```

Comment:
#c9c9c9 isn't a bad middleground and changing the links to #FFF is more than doable I'll submit that now.